nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/PutS3Object.java

```diff
@@ -177,10 +179,19 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor SERVER_SIDE_ENCRYPTION = new PropertyDescriptor.Builder()
+            .name("Server Side Encryption")
```

Comment:
I totally understand consistency over convention. If I'm being selfish, I'd say it is better to do it correctly *and* correct the other existing settings. The preferred and prescribed form is to use `.name()` for the unique value. See `EncryptContent` processor for a more complete example.